caffe2/perfkernels/CMakeLists.txt

```diff
@@ -42,6 +46,22 @@ if(CXX_AVX2_FOUND)
        "Caffe2_perfkernels_avx2_interface")
 endif()
 
+# We will only build the SVE perfkernel files if the compiler supports SVE
+# extensions.
+if(CXX_SVE_FOUND AND NOT MSVC)
```

Comment:
Can you add a comment about why not MSVC?

Reply:
Thanks for the catch. This is no longer necessary. Removed.